test/acceptance/features/claimForCluster.feature

```diff
@@ -0,0 +1,157 @@
+            namespace: primaza-system
+        spec:
+            environmentName: stage
+            clusterContextSecret: primaza-kw
+            applicationNamespaces:
+            - applications
+        """
+        And On Primaza Cluster "primaza-main", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: RegisteredService
+        metadata:
+          name: primaza-rsdb
+          namespace: primaza-system
+        spec:
+          constraints:
+            environments:
+            - stage
+          serviceClassIdentity:
+            - name: type
+              value: psqlserver
+            - name: provider
+              value: aws
+          serviceEndpointDefinition:
+            - name: host
+              value: mydavphost.io
+            - name: port
+              value: "5432"
+            - name: user
+              value: davp
+            - name: password
+              value: quedicelagente
+            - name: database
+              value: davpdata
+          sla: L3
+          """
+        And On Primaza Cluster "primaza-main", RegisteredService "primaza-rsdb" state will eventually move to "Available"
+        And On Worker Cluster "primaza-worker", Resource is created
+        """
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: stage-app
+          namespace: applications
+          labels:
+            app: stage-app
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: stage-app
+          template:
+            metadata:
+              labels:
+                app: stage-app
+            spec:
+              containers:
+              - name: bash
+                image: bash:latest
+                command: ["sleep","infinity"]
+        """
+        When On Primaza Cluster "primaza-main", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: ServiceClaim
+        metadata:
+          name: sc-test
+          namespace: primaza-system
+        spec:
+          serviceClassIdentity:
+          - name: type
+            value: psqlserver
+          - name: provider
+            value: aws
+          serviceEndpointDefinitionKeys:
+          - host
+          - port
+          - user
+          - password
+          - database
+          applicationClusterContext:
+            clusterEnvironmentName: primaza-worker
+            namespace: applications
+          application:
+            kind: Deployment
+            apiVersion: apps/v1
+            name: stage-app
+        """
+        Then On Primaza Cluster "primaza-main", the status of ServiceClaim "sc-test" is "Resolved"
+        And  On Worker Cluster "primaza-worker", the secret "sc-test" in namespace "applications" has the key "type" with value "psqlserver"
+        And  On Worker Cluster "primaza-worker", the service binding "sc-test" in namespace "applications" is bound
```

Comment:
if you want you can also check the content of the file mounted into the pod adapting the following step.

https://github.com/primaza/primaza/blob/346cadb510f35f8e824474df21fea1323add721f/test/acceptance/features/steps/kubernetescli.py#L620

Reply:
That's the functionality of ServiceBinding which we have tested. Checking the status of ServiceBinding is sufficient here.

Reply:
I agree